tests/e2e/cypress/e2e/combo-box.cy.js

```diff
@@ -0,0 +1,113 @@
+/* eslint no-unused-expressions: off, cypress/no-unnecessary-waiting: off */
+
+const WAIT_TIME = 2000;
+
+describe.skip("wx-combo-box tests", () => {
+  before(() => {
+    // Stub the ArcGIS API requests to use
+    // saved data
+    cy.intercept(
+      "https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/suggest*",
+      { fixture: "arc.query.json" }
+    ).as("arcSearch");
+    cy.intercept(
+      "https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/find*",
+      { fixture: "arc.query.items.json" }
+    ).as("arcSearchItems");
+
+    // Go to landing page
+    cy.visit("http://localhost:8080/");
+  });
+  it("Can find the combo-box element", () => {
+    cy.get("wx-combo-box")
+      .should("exist");
+    cy.get("wx-combo-box > input").should("exist");
+  });
+  describe("When typing into the input", () => {
+    beforeEach(() => {
+      // Go to landing page
+      // Stub the ArcGIS API requests to use
+      // saved data
+      cy.intercept(
+        "https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/suggest*",
+        { fixture: "arc.query.json" }
+      ).as("arcSearch");
+      cy.intercept(
+        "https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/find*",
+        { fixture: "arc.query.items.json" }
+      ).as("arcSearchItems");
+
+      // Go to landing page
+      cy.visit("http://localhost:8080/");
```

Comment:
Leftover cruft, good catch. Removed in https://github.com/weather-gov/weather.gov/pull/992/commits/213dd7821f0cd9eb4818e5a3c83a571cce707cf6. Note, however, that these tests are all set to skip currently because they are flaky for unknown reasons